pyproject.toml

```diff
@@ -136,6 +136,7 @@ sqlglot = "^10.1.3,<10.4"  # SQL generation doesn't match as >10.4 double quotes
 thrift-sasl = { version = "^0.4.3", optional = true }
 typeguard = "^2.13.3"
 typer = "^0.7.0"
+typing-extensions = "4.5.0"
```

Comment:
@gee-senbong Newer versions of typeguard does not seem to work in darwin env (seems to work in linux). Pinning this for now so we can investigate separately.